aggregator/config/config.ini

```diff
@@ -3,10 +3,10 @@
 
 [app]
 # Application hostname, overwritten by ENV $APP_HOST
-host=0.0.0.0
+host=localhost

```

Comment:
please don't modify default config, this is deployment specific and can be overwritten